### DISCUSSION TITLE
Nvm

### DISCUSSION THREAD

**Discussion by tahlequawesome** [mcguirepr89/BirdNET-Pi#713] [Q&A]
I asked about how to deal with emergency services being detected as an Eastern Screech-Owl, which is my most-frequent detection. I then figured out that's probably what the excluded species function is for. My bad.

**Comment by Scraggarax** [comment]
I unfortunately had to exclude it from my list. It was happening far too often with firetrucks specifically.

Also the trumpeter swan because of car horns.


Might have to do the yellow rail too as it detects the water dropping sound when it rains.

**Reply by ehpersonal38** [reply]
The new model might be better at preventing false positives, you can test it out if you want, see the discussions

**Comment by FirstTenor** [comment]
I keep getting a Boreal Owl being detected about 2 - 3 am most mornings.  The recordings sound very like library references and nothing like any other bird I have heard (light warbling ‘hoot’) ….. but I live in UK and they are not meant to be here!.

I also get an Eurasian Pygmy Owl at about 5am - but that sounds like the same motor bike each time.

I have seen other posts on false positives - it would be useful to be able to flag (and better - report) these…

**Reply by ehpersonal38** [reply]
The new model might be better at preventing false positives, you can test it out by updating (Tools -> System Controls -> Update) and setting the model to the new one (Tools -> Settings -> Model).
